app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
If you can confirm that hammer org command works correctly with your changes, that would be an indication that your change is the correct one. You may also wish to contact the apipie devs for clarification. (I am not positive how all the apipie doc extraction works.) You should also be able to see the docs on a running katello server.

Sorry for the questioning but insuring that katello continues to work with foreman changes is important. Thanks!